docs/docs/usage/streams/signature-verification.md

```diff
@@ -0,0 +1,57 @@
+---
+sidebar_position: 9
+---
+
+# Signature verification
+Authenticity and integrity of events published on a stream is guaranteed with digital signatures. All messages published to streams are cryptographically signed, which prevents tampering of messages and allows recipients to validate the message's publisher.
+
+:::info Practicalities
+- In the browser, every published message will typically need to be manually signed by metamask. 
+- When viewing a private (encrypted) stream in the browser, Metamask may ask for your signature. This is to verify that you are in the authorised set of the stream's subscribers.
+- Signing is instrumental to the protocol and cannot be turned off. 
+:::
+
+For more in depth analysis, see [Security - Data signing & verification](../../streamr-network/security/signing-and-verification.md).
+
+## Smart contract pub/sub (ERC-1271)
+Beside the typical externally owned account signing and verification, the Streamr protocol also supports smart contract signature verification, i.e. ERC1271 support. This feature allows the use of smart contracts that follow the ERC1271 spec to become signatories on streams in the Streamr Network.
+
+:::info Key points:
+- This feature is enabled on a per stream basis.
```

Comment:
> This feature is enabled on a per stream basis.

Just to clarify: a stream can simultaneously have both ERC-1271 contracts as publishers as well as normal publishers. I'm not sure if adding this clarification here would just add to confusion however. But just so we are internally aligned at least.

It's actually activated on a "per publishing session" basis when you pass the `erc1271contract` parameter to the `streamrClient.publish` method. Of course a prerequisite is that the ERC-1271 contract address has been given publish permission to the stream.

Reply:
One way to phrase this maybe (or is this confusing?):
- This feature is enabled on a per stream basis.
- This feature is activated on a per publishing session basis.

Reply:
Ah, yes. Good point - I'll rephrase this section to capture that nuance.